crates/dojo-core/src/world.cairo

```diff
@@ -24,6 +32,9 @@ mod World {
     #[event]
     fn WorldSpawned(address: ContractAddress, name: ShortString) {}
 
+    #[event]
+    fn WorldInitialized(address: ContractAddress, caller: ContractAddress) {}
```

Comment:
Hm this can be combined with `Spawned` I think

Reply:
Moving it to `WorldSpawned`